llvm/test/Transforms/SLPVectorizer/alternate-cmp-swapped-pred-parent.ll

```diff
@@ -1,5 +1,7 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; REQUIRES: target={{aarch64|x86_64}}-{{.*}}
```

Comment:
Did you check this works as expected?

With this patch merged, those tests are now considered 'unsupported' even when all targets are enabled.

You can check this by applying the patch below and observe that there are no test failures

```
diff --git a/llvm/test/Transforms/SLPVectorizer/alternate-cmp-swapped-pred-parent.ll b/llvm/test/Transforms/SLPVectorizer/alternate-cmp-swapped-pred-parent.ll
index a709af4cf858..1511dfbe37b6 100644
--- a/llvm/test/Transforms/SLPVectorizer/alternate-cmp-swapped-pred-parent.ll
+++ b/llvm/test/Transforms/SLPVectorizer/alternate-cmp-swapped-pred-parent.ll
@@ -6,7 +6,6 @@
 define void @test() {
 ; CHECK-LABEL: @test(
 ; CHECK-NEXT:  entry:
-; CHECK-NEXT:    [[CALL37:%.*]] = load i16, ptr poison, align 2
 ; CHECK-NEXT:    br label [[BB:%.*]]
 ; CHECK:       bb:
 ; CHECK-NEXT:    [[CALL:%.*]] = load i16, ptr poison, align 2
```

Reply:
yes, I checked and locally for me it worked fine
There are some buildbots failures as well, I'll try to figure out what's going on asap

Reply:
If it helps, the above is for ARM64 macOS

Reply:
@fhahn it should be fixed now https://github.com/llvm/llvm-project/pull/106531